shared/transforms/combinechecks.py

```diff
@@ -26,8 +29,8 @@ def _header(schema_version):
         <oval:product_name>python</oval:product_name>
         <oval:product_version>2.6.6</oval:product_version>
         <oval:schema_version>%s</oval:schema_version>
-        <oval:timestamp>2011-09-21T13:44:00</oval:timestamp>
-    </generator>''' % schema_version
+        <oval:timestamp>'''+str(timestamp)+'''</oval:timestamp>
```

Comment:
@thenefield still need to change ```'''+str(timestamp)+'''``` to ```%s``` before it can be merged, so the line should look like this:

```
<oval:timestamp>%s</oval:timestamp>
```